book.rb

```diff
@@ -0,0 +1,12 @@
+class Book
+  attr_accessor :title, :author
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+  end
+
```

Comment:
- [ ] Please make sure the `rentals` variable is initialized in the initializer, also defined in the `attr_accessor`, and add a method to add rentals.
  - You can consider the below code.
   
 ```ruby
def add_rental(person, date)
    Rental.new(date, self, person)
  end
```